### DISCUSSION TITLE
APU and electronic is not start!

### DISCUSSION THREAD

**Discussion by AlexandeKlikov** [flybywiresim/aircraft#4723] [Help]
Hello! I apologize in advance for my English! I'm desperate. The APU and all electronics do not want to start, when you press the "external power" key, there are no characteristic sounds, an empty click. There were attempts to delete everything from the Community folder, even reinstalled the simulator. That didn't solve the problem. With the release of the patch stable 6.1 was hoping that the problem would disappear. Nothing has changed. Cry of the soul, please help. Maybe something is in conflict with something?

**Comment by CptBayer** [comment]
Hi,

I experienced the same problem at cold and dark status with the experimental version. Click on the battery buttons and then external power button will be iluminated ... power is available ...

Greets Klaus

**Reply by ExampleWasTaken** [reply]
Just to prevent possible confusion here:
You are totally right: You have to first turn on the batteries, but the EXT PWR pb will be illuminated whenever EXT PWR is available. Even if the aircraft is completely powered down.

**Reply by AlexandeKlikov** [reply]
Initially, there is no battery voltage (0.00)-(0.00). There is no way to start it

**Comment by ExampleWasTaken** [comment]
Empty batteries can usually be solved by removing **ALL** content from your community folder, including the A32NX and re-installing it through the [official FlyByWire installer](https://api.flybywiresim.com/installer).
